source/agora/node/Ledger.d

```diff
@@ -247,7 +254,13 @@ public class Ledger
 
     public void writeBlockFile (ref Block block) @safe
```

Comment:
This function should probably take the File as a parameter. Opening a file costs performance, it should only be opened once, and this method should write to the file using the existing File instance.